Tests/SwiftSyntaxTest/TriviaTests.swift

```diff
@@ -69,4 +69,325 @@ class TriviaTests: XCTestCase {
     XCTAssertNotEqual(TriviaPiece.unexpectedText("e"), .unexpectedText("f"))
     XCTAssertNotEqual(TriviaPiece.unexpectedText("e"), .lineComment("e"))
   }
+
+  func testTriviaCommentValues() {
```

Comment:
I think it’s fine if we tailor `commentValue` to the kind of trivia that the parser produces. If we do that we can construct the `Trivia` from a string literal and have something like the following:

```swift
func assertCommentValue(
  _ trivia: Trivia,
  commentValue: String,
  file: StaticString = #filePath,
  line: UInt = #line
) {
  XCTAssertEqual(trivia.commentValue, commentValue, file: file, line: line)
}

func testExample() {
  assertCommentValue(
    "// line comment",
    commentValue: "line comment"
  )
}

```